rb/lib/selenium/webdriver/common/error.rb

```diff
@@ -77,11 +77,7 @@ class UnknownCommandError < WebDriverError; end
       # A command failed because the referenced element is no longer attached to the DOM.
       #
 
-      class StaleElementReferenceError < WebDriverError
-        def initialize(msg = '')
-          super("#{msg}; #{SUPPORT_MSG} #{URLS[:StaleElementReferenceError]}")
-        end
-      end
+      class StaleElementReferenceError < WebDriverError; end
```

Comment:
Was it intentional to include this change? The exception message was changed, but it seems unrelated?

Reply:
Hi @ioquatix, yes this was to keep consistency with my previous pr #14174